### PR TITLE
feat: expose all left org related endpoints

### PR DESCRIPTION
**Pull request by suevalov**
Finishes exposing all existing endpoints to a plain client.

### CODE REVIEW

File: lib/plain/endpoints/team-space-membership.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    `/organizations/${params.organizationId}/team_space_memberships/${params.teamSpaceMembershipId}`
+  )
+}
+
+export const getManyForOrganization = (
+  http: AxiosInstance,
+  params: GetOrganizationParams & QueryParams & { teamId?: string }
+) => {
+  const query = params.query || {}
+  if (params.teamId) {
```
Comment (anho) [inline lib/plain/endpoints/team-space-membership.ts:40]: I'd rather leave this magic in the organization api of the old client.

File: lib/entities/team-membership.ts [hunk 2]
```diff
@@ -69,25 +73,23 @@ export interface TeamMembership extends TeamMembershipProps, DefaultElements<Tea
...
-        })
-        .then((response) => wrapTeamMembership(http, response.data), errorHandler)
+      return endpoints.teamMembership
+        .update(http, getParams(raw), raw)
+        .then((data) => wrapTeamMembership(http, data))
     },
 
-    delete: function () {
+    delete: function del() {
       const raw = this.toPlainObject()
```
Comment (vbeleuta) [inline lib/entities/team-membership.ts:91]: i believe should be: `const raw = this.toPlainObject() as TeamMembershipProps`

**Review by vbeleuta — APPROVED** [review]
looks great 👍  just a small comment on team membership delete

**Review by anho — COMMENTED** [review]
Thanks for fixing `entities/team*` stuff.

File: lib/plain/endpoints/team-space-membership.ts (hunk 1 above)
Comment (suevalov) [reply lib/plain/endpoints/team-space-membership.ts:40]: I wouldn't say it's magic but rather a part of the how API works. We didn't leave any implementation details in the official client in other methods, so I'd rather be consistent here.

File: lib/entities/team-membership.ts (hunk 2 above)
Comment (suevalov) [reply lib/entities/team-membership.ts]: good call. fixed.

File: lib/plain/endpoints/team-space-membership.ts (hunk 1 above)
Comment (anho) [reply lib/plain/endpoints/team-space-membership.ts:40]: I think it's something different to what we provide in other places. But no strong opinion on this one.

**Review by anho — APPROVED** [review]

**Pull request merged**